setup.py

```diff
@@ -0,0 +1,16 @@
+from setuptools import setup, find_packages
+
+setup(
+    name='colin',
+    version='0.1',
+    packages=find_packages(),
+    install_requires=[
+        'Click',
```

Comment:
someone doesn't like argparse

Reply:
Yes..;-) Is it a big problem?

Reply:
I don't think so, let's wait what @phracek would think.